pkgs/development/androidndk-pkgs/androidndk-pkgs.nix

```diff
@@ -1,6 +1,7 @@
-{ lib, stdenv
+{ lib
 , makeWrapper
-, runCommand, wrapBintoolsWith, wrapCCWith
+, pkgs
+#, runCommand, wrapBintoolsWith, wrapCCWith
```

Comment:
Hmm I wonder of getting rid of `pkgs` but using `callPackage` down below might help with some issues.

Then again, I have to think I would have made it us `callPackage` originally if I could. Not sure.